### ISSUE TITLE
seqwr deleting test files

### ISSUE THREAD

**Issue by utdrmac** [akopytov/sysbench#44]
test_file's are being removed during the test and thus the test can't be performed.

This issue is NOT observed during any other modes (seqrewr, seqrd, rndrd, rndwr, nor rndrw)

```
root@pine2:/mnt/sysbench# sysbench --test=fileio --file-total-size=20G prepare
sysbench 0.5:  multi-threaded system evaluation benchmark

128 files, 163840Kb each, 20480Mb total
Creating files for the test...
Extra file open flags: 0
Creating file test_file.0
Reusing existing file test_file.1
Reusing existing file test_file.2
...
Reusing existing file test_file.126
Reusing existing file test_file.127
167772160 bytes written in 5.29 seconds (30.25 MB/sec).

root@pine2:/mnt/sysbench# ls -la | head
total 20971528
drwxr-xr-x 2 root root      4096 May  4 16:56 .
drwxr-xr-x 6 root root      4096 Apr 28 16:08 ..
-rw------- 1 root root 167772160 May  4 16:56 test_file.0
-rw------- 1 root root 167772160 May  4 15:29 test_file.1
-rw------- 1 root root 167772160 May  4 15:29 test_file.10
-rw------- 1 root root 167772160 May  4 15:29 test_file.100
-rw------- 1 root root 167772160 May  4 15:29 test_file.101
-rw------- 1 root root 167772160 May  4 15:29 test_file.102
-rw------- 1 root root 167772160 May  4 15:29 test_file.103

root@pine2:/mnt/sysbench# sysbench --test=fileio --file-total-size=20G --file-test-mode=seqwr --init-rng=on --max-time=1200 --max-requests=0 run
sysbench 0.5:  multi-threaded system evaluation benchmark

Running the test with following options:
Number of threads: 1
Random number generator seed is 0 and will be ignored

Extra file open flags: 0
128 files, 160Mb each
20Gb total file size
Block size 16Kb
Periodic FSYNC enabled, calling fsync() each 100 requests.
Calling fsync() at the end of test, Enabled.
Using synchronous I/O mode
Doing sequential write (creation) test
FATAL: Cannot open file 'test_file.0' errno = 2 (No such file or directory)
```

**Comment by akopytov** [comment]
I think I fixed it in 1.0 about a month ago. It is now backported to 0.5. Thanks.

**Comment by utdrmac** [comment]
Sweet. Seems to be working now! Thanks!
